mmedit/registry.py

```diff
@@ -62,3 +62,6 @@
 
 # manage optimizer wrapper
 OPTIM_WRAPPERS = Registry('optim_wrapper', parent=registry.OPTIM_WRAPPERS)
+
+# manage diffusers
+DIFFUSERS = Registry('diffuser')
```

Comment:
Do we really need a diffuser registry? 
What's the difference between diffuser and model?

Reply:
A diffusion model generally will contain a unet(network) and a diffuser(diffusers), and inherit from BaseModel.
A diffuser only take unet(network)'s output and compute ``x_t-1`` from ``x_t``. 

Since we can use ddpm/ddim/pndm diffuser to sample from ADM, we make diffuser a settable field in config, so we need a diffuser registry to build a diffuser in a algorithm class.

Reply:
A diffuser will also compute training loss related to its schedule.